README.md

```diff
@@ -19,3 +20,28 @@ quantitatively reason with them.
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+# Answer
+Median-of-three:
+1. On an average input, our chosen pivot is equally likely to be the $i^th$ smallest for any i = 1, 2, ... , n.
+2. With probability 1/3, our pivot will be from the middle $n$/2 elements – a good pivot.
+3. Any good pivot creates two partitions of size at most $3n/4$.
+4. We expect to pick one good pivot every three tries.
+5. Expected number of splits is at most 3 log<sub>(4/3)</sub> n ∈ Θ(log n).
+6. Θ($n$ log $n$) total comparisons.
+7. The probabilty of obatining a 3-to-1 split or better using median-of-three would be $11/16$ or about 68.75%
+
+First element:
+1. On an average input (i.e., a random order of n items), our chosen pivot is equally likely to be the ith smallest for any i = 1, 2, ..., n.
+2. With probability 1/n, our pivot will be the first element – a pivot that may not be as balanced as other methods.
+3. A pivot from the first element can create partitions of various sizes, depending on the input data.
+4. The probability of choosing a pivot that creates balanced partitions is lower compared to methods like median-of-three.
+5. Expected number of splits is determined by the distribution of the data, and it can be highly variable.
+6. The overall performance of quicksort with a first element pivot may vary depending on the input data and its distribution.
+7. Our expected probability of choosing a good pivot using the first element will be: $P(Good Pivot) = 1/n$.
+
+Upon revaluation the probabilities are not the same. Median-of-three will have a probability of selecting a good pivot of $11/16$ and selecting the first element will be $1/n$. Choosing the first element is simple but not optimal because as the input size increases the chances of finding a good pivot will decrease and could lead to unbalanced splits. 
```

Comment:
In class we said that with one candidate for a pivot you have a 50% chance of getting a good one. How are you getting $1/n$ now?